src/Automation/Automation.Test/ScenarioTests/AutomationTestRunner.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.TestFx;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.Network.Test.ScenarioTests
```

Comment:
The namespace here should be Microsoft.Azure.Commands.Automation.Test or Commands.Automation.Test